config/log4j2.properties

```diff
@@ -2,37 +2,26 @@ status = error
 name = LogstashPropertiesConfig
 
 appender.console.type = Console
-appender.console.name = STDOUT
-
-# Pattern Logging
-#
+appender.console.name = console
 appender.console.layout.type = PatternLayout
 appender.console.layout.pattern = [%d{ISO8601}][%-5p][%-25c] %m%n
-
 # JSON Logging
-#
 # appender.console.layout.type = JSONLayout
 # appender.console.layout.compact = true
 # appender.console.layout.eventEol = true
 
-# Rolling File Appender
-#
-#property.filename = /tmp/logstash/logstash.log
-#
-# appender.rolling.type = RollingFile
-# appender.rolling.name = RollingFile
-# appender.rolling.fileName = ${filename}
-# appender.rolling.filePattern = /tmp/logstash/logstash-%d{MM-dd-yy-HH-mm-ss}-%i.log.gz
-# appender.rolling.policies.type = Policies
-# appender.rolling.policies.time.type = TimeBasedTriggeringPolicy
-# appender.rolling.policies.time.interval = 2
-# appender.rolling.policies.time.modulate = true
-# appender.rolling.policies.size.type = SizeBasedTriggeringPolicy
-# appender.rolling.policies.size.size=100MB
-# appender.rolling.strategy.type = DefaultRolloverStrategy
-# appender.rolling.strategy.max = 5
-# appender.rolling.layout.type = PatternLayout
-# appender.rolling.layout.pattern = [%d{ISO8601}][%-5p][%-25c] %m%n
+appender.rolling.type = RollingFile
+appender.rolling.name = rolling
+appender.rolling.fileName = ${sys:ls.logs}/logstash.log
```

Comment:
@suyograo, Elasticsearch uses the node name as the name of this file. should we do the same?

I left it as `logstash.log` and `logstash-yyyy-MMM-ddd.log` because that is what was being passed in by the package distributions.

I know we now have a `setting('node.name')`, so I can do this as well.